layout/templates/lakeWarming.mustache

```diff
@@ -5,12 +5,8 @@
 </div>
 
 <div id="lakeWarming" class="center">
-	  <div class="list">
-	    <li>{{lakeWarming-l1}}</li>
-	    <li>{{lakeWarming-l2}}</li>
-	    <li>{{lakeWarming-l3}}</li>
-	    <li>{{lakeWarming-l4}}</li>
-	    <li>{{lakeWarming-l5}}</li>
-	  </div>
```

Comment:
yeah, I like this better, though the <li> stuff should be wrapped in <ul> or <ol>